src/components/Footer.tsx

```diff
@@ -50,7 +49,7 @@ export const Footer: React.FC<{}> = () => (
     <table style={tableStyle}>
         <tr>
             <td style={tdPadding}>
-                <table style={tableWrapper}>
+                <table bgcolor={palette.neutral[20]} style={tableWrapper}>
```

Comment:
It might be worth commenting this or documenting the motivation somewhere? I can imagine future people working on this won't understand why bgcolor is handled as an attribute rather than in the styles here.